### PR TITLE
Added support for ChibiOS clients

### PR DESCRIPTION
**Pull request by stertingen**

### CODE REVIEW

File: rosserial_chibios/src/rosserial_chibios/make_libraries.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+rospack = rospkg.RosPack()
+
+# copy ros_lib stuff in
+rosserial_chibios_dir = rospack.get_path(THIS_PACKAGE)
+shutil.copytree(rosserial_chibios_dir+"/src/ros_lib", path+"/ros_lib")
+rosserial_client_copy_files(rospack, path+"/ros_lib/")
+
+# generate messages
+rosserial_generate(rospack, path+"/ros_lib", ROS_TO_EMBEDDED_TYPES)
+
```
Comment (mikepurvis) [inline rosserial_chibios/src/rosserial_chibios/make_libraries.py]: I know this is just a c&p of similar other files in this repo, but please lose the trailing line and add spaces around the `+` operators above (or better yet, switch them to be `os.path.join`).

**Review by mikepurvis — CHANGES_REQUESTED** [review]

File: rosserial_chibios/src/rosserial_chibios/make_libraries.py (hunk 1 above)
Comment (stertingen) [reply rosserial_chibios/src/rosserial_chibios/make_libraries.py]: So be it.

`os.path.join` strips trailing slashes from the path; which breaks further generation of `ros_lib`.

There'll be a PR to migrate everything to `os.path.join` in a minute.

Comment (mikepurvis) [reply rosserial_chibios/src/rosserial_chibios/make_libraries.py]: Ha, appreciated.

**Review by mikepurvis — APPROVED** [review]

**Comment by mikepurvis** [comment]
Retargeting this to `noetic-devel` and marking for backport consideration.

**Comment by mikepurvis** [comment]
Looks good, thanks!

**Pull request merged**